how-to/what-is-vm-cache-and-how-do-I-use-it.md

```diff
@@ -0,0 +1,31 @@
+# What Is VMCache and How To Enable It
```

Comment:
Please could you add a table of contents at the top?

Reply:
> Please could you add a table of contents at the top?

Added.